2.0/m_antirandom.cpp

```diff
@@ -564,52 +582,27 @@ class ModuleAntiRandom : public Module
 			}
 
 			/* Check consonants */
-			switch (*s)
+			if (strchr("bcdfghjklmnpqrstvwxz", c))
 			{
-				case 'b':
-				case 'c':
-				case 'd':
-				case 'f':
-				case 'g':
-				case 'h':
-				case 'j':
-				case 'k':
-				case 'l':
-				case 'm':
-				case 'n':
-				case 'p':
-				case 'q':
-				case 'r':
-				case 's':
-				case 't':
-				case 'v':
-				case 'w':
-				case 'x':
-				case 'z':
-					consonants++;
-					break;
-				default:
-					if (consonants > highest_consonants)
-						highest_consonants = consonants;
-					consonants = 0;
-					break;
+				consonants++;
+			}
+			else
+			{
+				if (consonants > highest_consonants)
+					highest_consonants = consonants;
+				consonants = 0;
 			}
 
 			/* Check vowels */
-			switch (*s)
+			if (strchr("aeiou", c))
 			{
-				case 'a':
-				case 'e':
-				case 'i':
-				case 'o':
-				case 'u':
-					vowels++;
-					break;
-				default:
-					if (vowels > highest_vowels)
-						highest_vowels = vowels;
-					vowels = 0;
-					break;
+				vowels++;
+			}
+			else
+			{
+				if (vowels > highest_vowels)
+					highest_vowels = vowels;
+				vowels = 0;
```

Comment:
here is the code: https://gist.github.com/linuxdemon1/775890a8924b3f0972a47a1ce705ed8f